tests/network-tests/package.json

```
@@ -4,13 +4,16 @@
   "license": "GPL-3.0-only",
   "scripts": {
     "build": "tsc --build tsconfig.json",
-    "test": "tap --files ts-node/register src/constantinople/tests/proposals/*Test.ts",
-    "test-migration": "tap --files src/rome/tests/romeRuntimeUpgradeTest.ts --files src/constantinople/tests/electingCouncilTest.ts",
+    "test": "tap --files ts-node/register src/nicaea/tests/proposals/*Test.ts --files ts-node/register src/nicaea/tests/workingGroup/*Test.ts -T",
+    "test-migration-constantinople": "tap --files src/rome/tests/romeRuntimeUpgradeTest.ts --files src/constantinople/tests/electingCouncilTest.ts -T",
+    "test-migration-nicaea": "tap --files src/constantinople/tests/proposals/updateRuntimeTest.ts --files src/nicaea/tests/electingCouncilTest.ts -T",
+    "debug": "tap --files src/nicaea/tests/workingGroup/workerApplicationRejectionCaseTest.ts -T",
     "lint": "tslint --project tsconfig.json",
     "prettier": "prettier --write ./src"
   },
   "dependencies": {
-    "@constantinople/types": "./types",
+    "@constantinople/types@npm:@joystream/types": "^0.10.0",
+    "@nicaea/types": "./types",
```

```suggestion
    "@nicaea/types": "link:../../types",
```
Link protocol works better, as it will be a relative path from the package.json
Secondly by linking we avoid need to manually remove project-root/node_modules/@nicaea before running yarn again in the root worksapce after making changes in the types library

---

Applied